crates/turbopack-ecmascript/src/transform/server_to_client_proxy.rs

```diff
@@ -52,7 +53,13 @@ pub fn create_proxy_module(transition_name: &str, target_import: &str) -> Progra
                 })],
                 src: box target_import.into(),
                 type_only: false,
-                asserts: None,
+                asserts: Some(box ObjectLit {
+                    span: DUMMY_SP,
+                    props: vec![PropOrSpread::Prop(box Prop::KeyValue(KeyValueProp {
+                        key: PropName::Ident(Ident::new("turbopackHelper".into(), DUMMY_SP)),
```

Comment:
Should this be named `__turbopackHelper` juuuust in case?

`"turbopackHelper"` should move to a `&'static str` constant that's shared between this module and `is_turbopack_helper_module_decl`.